Automattic/wp-calypso
# Plugin Management: enable new plugin management on Jetpack production env for production release

[DO-NOT-MERGE]

#### Proposed Changes

This PR enables new plugin management features on Jetpack production env

#### Testing Instructions

Verify feature `jetpack/plugin-management` & section `jetpack-cloud-plugin-management` is set to true in `jetpack-cloud-production.json`

#### Pre-merge Checklist

Complete applicable items on this checklist **before** merging into the trunk. Inapplicable items can be left unchecked.

The PR author and reviewer are responsible for completing the checklist.

- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes? Not required

Related to 1202518759611394-as-1202979359232050
